pulsar/connection/connection.go

```diff
@@ -75,11 +77,24 @@ func (*Factory) NewManager(settings map[string]interface{}) (connection.Manager,
 		}
 	}
 
+	customLogger := plogger.NewLoggerWithLogrus(logrus.StandardLogger())
```

Comment:
As we are using in flogo env,  I think it'd better have a flogo logger wrapper on this and use flogo logger.  We don't want to use logrus in flogo to have 2 different logger/formatt.

Reply:
In that case, you don't need to get the log level and set log level.  All managed by flogo engine

Reply:
Third party's library generates redundant/informative logs at Info level and so having same logger for Flogo and third party woudn't be helpful. So we want to get more logs from thirdparty library only when user running Flogo at Debug level for debugging perspective. For Info or any other level we configuring thirdparty logger at ERROR level.

Reply:
If I am not wrong the lib just requires an interface of `log.Logger`.   I meant to create flogo pulsar own logger which implements log.Logger and wrapper from flogo logger.

Reply:
updated code with zapwrapper logger. Done